index.php

```diff
@@ -3,31 +3,52 @@
 require('../../config.php');
 require_once("lib.php");
 
+$show_overview = !isset($_GET['id']);
+$context = context_user::instance($USER->id);
+$cansee_overview = has_capability('report/courseoverview:view', $context);
```

Comment:
I will ask @nscbrandb about their permission management in moodle

Reply:
He suggested to create an own permission: https://docs.moodle.org/dev/NEWMODULE_Adding_capabilities

Reply:
Added "mod/miquiz:overview" permission which inherits from "report/courseoverview:view"

Reply:
@hpawe01 should we document the permissions in the readme.md or in the manual?

Reply:
> @hpawe01 should we document the permissions in the readme.md or in the manual?

As it is a user related information we should put it in the manual, together with the information about the new full activity overview